jobs/director/templates/director.yml.erb.erb

```diff
@@ -189,6 +189,12 @@ if p('blobstore.provider') == "s3"
    params['blobstore']['options']['s3cli_path'] = "/var/vcap/packages/s3cli/bin/s3cli"
 end
 
+if p('blobstore.provider') == "gcs"
```

Comment:
If I had to guess, I'd say it would error out due to an empty bucket name. Running an adapted BATs, which I've yet to, would probably confirm this.